test/acceptance/test/add_tests.go

```diff
@@ -205,72 +205,72 @@ var _ = Describe("Weave GitOps App Add Tests", func() {
 		})
 	})
 
-	It("Verify that gitops can deploy and remove a gitlab app after it is setup with an empty repo initially", func() {
-		var repoAbsolutePath string
-		private := true
-		tip := generateTestInputs()
-		appName := tip.appRepoName
-		var appRemoveOutput *gexec.Session
+	// It("Verify that gitops can deploy and remove a gitlab app after it is setup with an empty repo initially", func() {
```

Comment:
:heavy_check_mark:  You can just put a `P` or `X` in front of an `It`, `Context`, etc. to make it get skipped. See "Pending Specs" here: 

https://onsi.github.io/ginkgo/